src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,40 @@
+package blackjack.domain
+
+class Player(
+    val name: String,
+    private val hand: Hand = Hand(),
+    var state: State = State.HIT,
+) {
+    init {
+        updateState()
+    }
+
+    fun getCardList(): List<Card> = hand.cards
+
+    fun canDraw(): Boolean =
+        getScore() <= BlackjackRule.TARGET_SCORE && state == State.HIT
+
+    fun draw(deck: Deck) {
+        hand.add(card = deck.pop())
+        updateState()
+    }
```

Comment:
```suggestion
    fun draw(deck: Deck) {
        receiveCard(deck.pop())
        updateState()
    }

    private fun receiveCard(card: Card) {
        hand.add(card)
    }
```

음.. 이건 개인 취향일 수도 있는데요~  hand.add(card = deck.pop())를 함수화한다면, 좀 더 함수를 정돈되게 만들 수 있어요. 😁